plugins/restful/RestfulEntityBase.php

```diff
@@ -1220,24 +1241,15 @@ public function publicFieldsInfo() {
   /**
    * {@inheritdoc}
    */
-  public function getPublicFields() {
-    if ($this->publicFields) {
-      // Return early.
-      return $this->publicFields;
-    }
-
-    // Get the public fields that were defined by the user.
-    $public_fields = $this->publicFieldsInfo();
-
+  protected function addDefaultValuesToPublicFields(array $public_fields = array()) {
+    $public_fields = parent::addDefaultValuesToPublicFields($public_fields);
     // Set defaults values.
     foreach (array_keys($public_fields) as $key) {
-      // Set default values.
+      // Set default values specific for entities.
       $info = &$public_fields[$key];
       $info += array(
         'access_callbacks' => array(),
-        'callback' => FALSE,
```

Comment:
not dropping, it's moving to the parent class. I'm trying to refactor a bit getPublicFields/ setPublicFields -- so it always adds defaults values to the configuration array (for easier testing).